source/server/http_dynamic_delay_filter.cc

```diff
@@ -13,9 +13,10 @@ namespace Nighthawk {
 namespace Server {
 
 HttpDynamicDelayDecoderFilterConfig::HttpDynamicDelayDecoderFilterConfig(
-    const nighthawk::server::ResponseOptions& proto_config, Envoy::Runtime::Loader& runtime,
-    const std::string& stats_prefix, Envoy::Stats::Scope& scope, Envoy::TimeSource& time_source)
-    : FilterConfigurationBase(proto_config, "dynamic-delay"), runtime_(runtime),
+    const nighthawk::server::DynamicDelayConfiguration& proto_config,
+    Envoy::Runtime::Loader& runtime, const std::string& stats_prefix, Envoy::Stats::Scope& scope,
+    Envoy::TimeSource& time_source)
+    : FilterConfigurationBase(proto_config.response_options(), "dynamic-delay"), runtime_(runtime),
```

Comment:
What is the value of inheriting from FilterConfigurationBase. I think this is adding more complexity than needed -  this config should be or inherit from fault.proto since the the filer itself inherits from it.

Reply:
Addressing in a followup refactor